### PR TITLE
Modified the logging for specific tests and runs

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Added a message for each run of the framework which would help to identify the specific runs.

Fixes : #185

Signed-off-by: aujjwal-redhat <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
Snaps of log files for specific tests
Screenshot from 2021-04-12 11-49-17
Screenshot from 2021-04-12 11-49-23
Screenshot from 2021-04-12 11-49-36

**Comment by srijan-sivakumar** [comment]
@aujjwal-redhat , maybe something a little more informative, Rather than just saying "This is a new test run!", we can print, `Starting Test Run : Timestamp.`

And once the Test finishes another log saying `Test run finished : Timestamp`.

**Comment by srijan-sivakumar** [comment]
Also, logger init actually comes when a new TC starts..so a little misleading that the old test run finishes after logger initialization.

**Comment by nik-redhat** [comment]
> @aujjwal-redhat , maybe something a little more informative, Rather than just saying "This is a new test run!", we can print, `Starting Test Run : Timestamp.`
> 
> And once the Test finishes another log saying `Test run finished : Timestamp`.

Is it possible to log the test name as well? More user-friendly.

**Comment by srijan-sivakumar** [comment]
> > @aujjwal-redhat , maybe something a little more informative, Rather than just saying "This is a new test run!", we can print, `Starting Test Run : Timestamp.`
> > And once the Test finishes another log saying `Test run finished : Timestamp`.
> 
> Is it possible to log the test name as well? More user-friendly.

+1

File: support/relog.py [hunk 1]
```diff
@@ -34,6 +34,13 @@ def init_logger(self, mname: str, log_file_path: str,
         log_file_handler = logging.handlers.WatchedFileHandler(log_file_path)
         log_file_handler.setFormatter(log_format)
         self.logger.addHandler(log_file_handler)
+        self.logger.info('''
```
Comment (srijan-sivakumar) [inline support/relog.py:37]: As mentioned in the comments. If this can be modified to a little more user friendly form.

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]

**Comment by aujjwal-redhat** [comment]
> @aujjwal-redhat , maybe something a little more informative, Rather than just saying "This is a new test run!", we can print, `Starting Test Run : Timestamp.`
> 
> And once the Test finishes another log saying `Test run finished : Timestamp`.

@srijan-sivakumar There is already a time stamp.

```bash
[2021-04-12 11:39:43,046] INFO [relog - 37:init_logger] - 
         ===================================
        ||                                 ||
        ||  This is a new run!!            || 
        ||                                 ||
         ===================================
```

**Comment by srijan-sivakumar** [comment]
> > @aujjwal-redhat , maybe something a little more informative, Rather than just saying "This is a new test run!", we can print, `Starting Test Run : Timestamp.`
> > And once the Test finishes another log saying `Test run finished : Timestamp`.
> 
> @srijan-sivakumar There is already a time stamp.
> 
> ```shell
> [2021-04-12 11:39:43,046] INFO [relog - 37:init_logger] - 
>          ===================================
>         ||                                 ||
>         ||  This is a new run!!            || 
>         ||                                 ||
>          ===================================
> ```

Ack, let's remove this log saying logger init..We can start with TC starts running and add the name of the TC along with the volume type, Something of the form, <tc_name>-<volume_type>

**Comment by aujjwal-redhat** [comment]
Snaps after changes:
Screenshot from 2021-04-12 13-02-35
Screenshot from 2021-04-12 13-03-05
Screenshot from 2021-04-12 13-04-09

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**